sysutils/pfSense-pkg-AutoConfigBackup/files/usr/local/pkg/autoconfigbackup.inc

```diff
@@ -150,13 +178,15 @@ function test_connection($post) {
 function upload_config($reasonm = "") {
 	global $config, $g, $input_errors;
 
-	// Do nothing when booting
+	// Do nothing when booting or when not enabled
 	if (function_exists("platform_booting")) {
 		if (platform_booting()) {
 			return;
 		}
 	} elseif ($g['booting']) {
 		return;
+	} elseif (!acb_enabled()) {
```

Comment:
That can go. That function was added on 2.2, and this code wouldn't ever touch anything that old so it can be cleaned up.